mfthomps/CyberCIEGE-port-to-Unity3d
# Create make more characters (resolves #227)

Closes #227 
To test:

Run the Introduction scenario and observe that Moe and Curly wear shirts of different color.
